resources/5/ajax.js

```diff
@@ -0,0 +1,223 @@
+const url = "http://gamf.nhely.hu/ajax2/";
+const code = "VE22CSabc123";
+document.getElementById("code").innerHTML = "code=" + code;
+
+async function read() {
+  try {
+    let response = await fetch(url, {
+      method: 'post',
+      cache: 'no-cache',
+      headers: {
+        'Content-Type': 'application/x-www-form-urlencoded',
+      },
+      body: "code=" + code + "&op=read"
+    });
+    let data = await response.text();
+    try {
+      data = JSON.parse(data);
+    } catch (jsonError) {
+      console.error("Hiba: A válasz nem JSON formátumú!", data);
+      return;
+    }
+
+    let list = data.list;
+    let sum = 0;
+    let max = 0;
+
+    let str = "<h3>Adatok lekérése</h3>";
+    str += "<p>Rekordok száma: " + data.rowCount + "</p>";
+    str += "<p>Utolsó max " + data.maxNum + " records:</p>";
+    str += "<table><tr><th>id</th><th>\tname</th><th>height</th><th>weight</th><th>code</th></tr>";
+
+    for (let i = 0; i < list.length; i++) {
+      let height = parseFloat(list[i].height);
+      if (!isNaN(height)) {
+        sum += height;
+        if (height > max) {
+          max = height;
+        }
+      }
+      str += `<tr>
+              <td>${list[i].id}</td>
+              <td>${list[i].name}</td>
+              <td>${list[i].height}</td>
+              <td>${list[i].weight}</td>
+              <td>${list[i].code}</td>
+          </tr>`;
+    }
+    str += "</table>";
+
+    let readDiv = document.getElementById("readDiv");
+    if (readDiv) {
+      readDiv.innerHTML = str;
+    }
+    else {
+      console.error("Hiba: readDiv nem található az oldalon!");
```

Comment:
Console log messages should be in English, it is for the developers not for the Users